app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::GreetingsController < ApplicationController
+  before_action :set_api_v1_greeting, only: %i[show update destroy]
+
+  # GET /api/v1/greetings
+  def index
+    random_greeting = Greeting.order('RANDOM()').first
+    render json: random_greeting
+  end
+
```

Comment:
Fantastic job on working on your API 👏🏾💯
The API endpoint works as expected and returns a random greeting ✔️ 

![image](https://github.com/zamcham/hello-rails-back-end/assets/38088818/03851bfe-bc44-4735-a0b0-03d0e49f8f84)